grin/src/Pipeline/Pipeline.hs

```diff
@@ -257,6 +257,8 @@ data PipelineOpts = PipelineOpts
   , _poLintOnChange :: Bool
   , _poTypedLint :: Bool -- Run HPT before every lint
   , _poSaveBinary :: Bool
+  , _poRuntimeC :: FilePath
```

Comment:
Why are `runtime.c` or `primop.c` sources special?
I'd rater store a list of c source files in general.

Reply:
As we dedicated them to be special before. I also like the list idea better. Chaging...